xwiki-platform-core/xwiki-platform-tour/xwiki-platform-tour-ui/src/main/resources/TourCode/StepSheet.xml

```diff
@@ -89,7 +89,12 @@
       &lt;/dl&gt;
     #else
       &lt;dl&gt;
-        &lt;dt&gt;&lt;label #if($isEditing)for="TourCode.StepClass_0_${prop.name}"#end&gt;$services.localization.render("${class.name}_${prop.name}")&lt;/label&gt;&lt;/dt&gt;
+    &lt;dt&gt;&lt;label #if($isEditing)for="TourCode.StepClass_0_${prop.name}"#end&gt;$services.localization.render("${class.name}_${prop.name}")&lt;/label&gt;
+    ## Add a hint for the field if there's one defined in the translations.
+    #if($!services.localization.render("${class.name}_${prop.name}.hint") != "${class.name}_${prop.name}.hint")##
```

Comment:
You can also use ``$services.localization.get()`` to check if a key is translated, see https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-notifications/src/main/resources/templates/mentions/mention.vm#L85-L89 .